src/org/thoughtcrime/securesms/ProfileSettingsFragment.java

```diff
@@ -205,6 +208,9 @@ else if(contactId>DcContact.DC_CONTACT_ID_LAST_SPECIAL) {
 
   public void onAddMember() {
     DcChat dcChat = dcContext.getChat(chatId);
+    if (!dcChat.canSend()) {
+      return;
+    }
```

Comment:
I think we should hide this button, too, instead of just doing nothing when it's clicked